ObjectsComparer/ObjectsComparer/Attributes/ComparisonAttribute.cs

```diff
@@ -0,0 +1,41 @@
+﻿// ***********************************************************************
+// Assembly         : Objects Comparer
+// Author           : Ankur Kumar Gupta
+// Created          : 24-Feb-2022
+// ***********************************************************************
+
+namespace ObjectsComparer.Attributes
+{
+  using System;
+  using ObjectsComparer.Enums;
+
+  /// <summary>
+  /// Class is used to specify whether the element on which it is applied will have comparison effect
+  /// </summary>
+  [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field)]
+  public sealed class ComparisonAttribute : Attribute
```

Comment:
Consider renaming to something like IgnoreInComparison

Reply:
Ok. That won't be a problem.